src/replay.rs

```diff
@@ -89,27 +89,30 @@ impl Arbiter for AddressArbiter {
     fn decide(&mut self, packet: &IpPacket) -> Action {
         const CONN_END: TcpFlags = TcpFlags::RST.union(TcpFlags::FIN);
 
-        // Check if this is a packet that should be sent in the replay
-        if self.host == packet.dest && self.client == packet.source {
-            // Has the right destination and source, now check that actually data was sent
-            if packet.payload.flags.contains(TcpFlags::PSH) {
-                return Action::Send(None);
-            }
-            // Check if connection got reset and break if it did
-            if packet.payload.flags.intersects(CONN_END) {
-                return Action::Break;
+        // If the packet is not a TCP packet just skip it
+        if let DataType::TCP(ref payload) = packet.payload {
```

Comment:
no support for udp in the replayer?

Reply:
That is kinda just an example class. I guess I can add one for UDP as well